### DISCUSSION TITLE
iframe csp

### DISCUSSION THREAD

**Discussion by josh900** [Budibase/budibase#4572] [Help]
Is there any way to modify the CSP for NGINX (I think), with the cli hosting method?

I am trying to embed another webpage I own into one of my apps, and need to allow for the iframe.

**Comment by akshayharshan** [comment]
hello @josh900,
did you figure out how to add iframe in budibase.

**Comment by akshayharshan** [comment]
I got the answer to the question actually, they enabled the "add_header  X-Frame-Options "SAMEORIGIN" always;" in the Nginx configuration of budibase that means it wouldn't allow iframe.  so we need to change the line to "proxy_hide_header X-Frame-Options;" and reload the Nginx, but the problem here is once you down the docker and then up it will disappear so I think it is better to move the proxy to your custom one by copy and pasting all the configuration that they had mentioned and change the above-mentioned line, hope it would work. plz, comment if anybody has other plans.
